io/net/bonding.py

```diff
@@ -319,6 +320,8 @@ def tearDown(self):
         set the initial state
         '''
         self.bond_remove("local")
+        self.log.info("Removing bonding module")
```

Comment:
Like you have done for peer, you can remove the module for host also in in `bond_remove()`